src/NServiceBus.Testing.Fakes/obsoletes.cs

```diff
@@ -0,0 +1,30 @@
+namespace NServiceBus.Testing
+{
+    using System;
+    using System.Threading.Tasks;
+    
+    public partial class TestableInvokeHandlerContext
+    {
+        /// <summary>
+        /// Moves the message being handled to the back of the list of available
+        /// messages so it can be handled later.
+        /// </summary>
+        [ObsoleteEx(
+            Message = "HandleCurrentMessageLater has been deprecated.",
+            RemoveInVersion = "8",
+            TreatAsErrorFromVersion = "7")]
+        public Task HandleCurrentMessageLater()
+        {
+            throw new NotSupportedException("HandleCurrentMessageLater has been deprecated and will be removed in NServiceBus.Core Version 8.");
```

Comment:
If we're moving things into an obsoletes.cs, I think it makes sense to throw `NotImplementedException` instead, like we always do: https://github.com/Particular/NServiceBus/blob/4cf52808635bf9f0a49b461381e800a84fe59384/src/NServiceBus.Core/obsoletes.cs#L48

You should also remove all the code comments, and disable the warning:
https://github.com/Particular/NServiceBus/blob/4cf52808635bf9f0a49b461381e800a84fe59384/src/NServiceBus.Core/obsoletes.cs#L5

Reply:
Neither of these have been done yet.